### DISCUSSION TITLE
Welcome to powerbi-php-sample Discussions!

### DISCUSSION THREAD

**Discussion by attilaersek** [attilaersek/powerbi-php-sample#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Moris1** [comment]
Hi,

my goal is to call getRevenueCenterDimensions endpoint of Power BI.
https://docs.oracle.com/en/industries/food-beverage/back-office/20.1/biapi/op-bi-v1-orgidentifier-getrevenuecenterdimensions-post.html

is that possible with your class?

**Reply by attilaersek** [reply]
It's been years since I've created this sample so there might be better solutions out there, but I think it should work. You just need to revisit the powerbi.php file and check how it constructs the api call.
